diagnostic.md

```diff
@@ -54,6 +66,21 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+.big{
+  ...;
+}
+
+#contentPane{
+  ...;
+}
+
+.important .red{
+  ...;
+}
+
+div p {
```

Comment:
The space is a descendent again. So for child/parent relationship, its `div > p`